source/modules/MEI.shared.xml

```diff
@@ -6548,66 +6548,58 @@
         <rng:optional>
           <rng:choice>
             <rng:group>
-              <rng:choice>
-                <rng:ref name="model.respLikePart"/>
-                <rng:ref name="respStmt"/>
-              </rng:choice>
+              <rng:ref name="mei_model.respLikePart"/>
```

Comment:
```suggestion
              <rng:ref name="model.respLikePart"/>
```

Reply:
I think the prefix`mei_` should be removed here and also in the below references

Reply:
Yes, they should be removed.